sunpy/net/helioviewer.py

```diff
@@ -61,12 +92,10 @@ def get_closest_image(self, date, **kwargs):
             (Optional) measurement name
```

Comment:
I don't think we should label them optional anymore. 

Do you know what happens if you only pass in a date to the function?